spec/models/post_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: posts
+#
+#  id         :bigint           not null, primary key
+#  body       :text             not null
+#  status     :integer          default("public"), not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint
+#
+# Indexes
+#
+#  index_posts_on_user_id  (user_id)
+#
+require 'rails_helper'
+
+RSpec.shared_examples 'Post' do
+  it 'expects post to be valid' do
+    expect(post).to be_valid
```

Comment:
it's more clean to use `subject` here, because shared example implies that in most cases you run spec in it for subject
and since you use subject, you can rewrite this spec to `is_expected.to be_valid`